tests/model/api/test_http_api_generator.py

```diff
@@ -325,6 +326,75 @@ def test_basic_route53(self):
         self.assertIsNotNone(route, None)
         self.assertEqual(domain.DomainNameConfigurations[0].get("EndpointType"), "REGIONAL")
 
+    @patch("boto3.session.Session.region_name", "eu-west-2")
```

Comment:
nit: we tend to prefer regular transform tests (i.e. check a SAM template transforms into an expected CloudFormation template), as the tests are more thorough (as technically others parts of the transform logic could transform the template as well), they're less coupled to implementation details, and they go through other tests as well (specifically, schema tests)

Reply:
No worries - I'll remove these. Thanks!